src/ShopArticleReviewBundle/objects/WebModules/MTPkgShopArticleReviewCore/MTPkgShopArticleReviewCore.class.php

```diff
@@ -100,9 +102,29 @@ protected function GetPkgCommentModuleConfiguration()
             $oPkgCommentModuleConfig->fieldNumberOfCommentsPerPage = 0;
             $oPkgCommentModuleConfig->fieldGuestCommentAllowed = !$oModuleConfiguration->fieldAllowWriteReviewLoggedinUsersOnly;
             $oPkgCommentModuleConfig->fieldPkgCommentTypeId = $this->GetCommentTypeId();
+
+            /**
+             * @psalm-suppress InvalidPropertyAssignmentValue
+             * @FIXME field is `bool` but assigned `int` - This could yield unwanted behaviour, especially with strict checks
+             */
             $oPkgCommentModuleConfig->fieldUseSimpleReporting = 1;
+
+            /**
+             * @psalm-suppress InvalidPropertyAssignmentValue
+             * @FIXME field is `bool` but assigned `int` - This could yield unwanted behaviour, especially with strict checks
+             */
             $oPkgCommentModuleConfig->fieldShowReportedComments = 0;
+
+            /**
+             * @psalm-suppress UndefinedPropertyAssignment
+             * @FIXME Does `fieldCountShowReviews` exist?
+             */
             $oPkgCommentModuleConfig->fieldCountShowReviews = $oModuleConfiguration->fieldCountShowReviews;
```

Comment:
At least in my installation, `fieldCountShowReviews` and `fieldAllowReportComments` did not exist